rosdep/base.yaml

```diff
@@ -69,6 +69,9 @@ apr:
   macports: [apr, apr-util]
   opensuse: [libapr1, libapr-util1]
   rhel: [apr-devel, apr-util]
+  slackware:
+    slackpkg:
```

Comment:
So slackpkg is the default package manager for Slackware. There is a limitation however - it works only with packages which come with Slackware by default. So it is mostly used to update and install (in case you did not install during OS installation) packages from the main Slackware tree. 

I have chosen sbotools as the default installer for the rules in rosdep. Sbotools is installer which can install packages from slackbuilds.org - a third party repository for slackware packages which has been approved by the Slackware team. 

Note that packages which come with the official Slackware OS will not be hosted on slackbuilds.org. Hence some packages in the rules above need to use slackpkg, others - sbotools.